Shared/AppConstants.swift

```diff
@@ -38,9 +38,13 @@ public struct KeychainKey {
 }
 
 public struct AppConstants {
-    public static let IsRunningTest = NSClassFromString("XCTestCase") != nil || ProcessInfo.processInfo.arguments.contains(LaunchArguments.Test)
+    public static let isRunningTest = NSClassFromString("XCTestCase") != nil
+    || AppConstants.isRunningUITests
+    || AppConstants.isRunningPerfTests
 
-    public static let IsRunningPerfTest = NSClassFromString("XCTestCase") != nil || ProcessInfo.processInfo.arguments.contains(LaunchArguments.PerformanceTest)
+    public static let isRunningUITests = ProcessInfo.processInfo.arguments.contains(LaunchArguments.Test)
+
+    public static let isRunningPerfTests = ProcessInfo.processInfo.arguments.contains(LaunchArguments.PerformanceTest)
```

Comment:
@isabelrios I think `isRunningPerfTests` is used in only at the same time of `isRunningUITests` and I was wondering if instead we could only used `isRunningUITests`? In other words, is `LaunchArguments.PerformanceTest` still needed?

Reply:
We added that LaunchArgument when we created the performance tests, that said, I don't see it is doing something in terms of configuring the build or so prior to the tests. I think it is not needed and can be removed (it should be removed also in the test suite that is using it: https://github.com/mozilla-mobile/firefox-ios/blob/4c0fdc1f59f174a1915c171727374462c96426d1/Tests/XCUITests/TabsPerformanceTests.swift#L14) 
I will keep an eye on those tests in case I'm missing something for which we added it